Modules/Sources/SFHFKeychainUtils/include/SFHFKeychainUtils.h

```diff
@@ -27,7 +27,7 @@
 //  OTHER DEALINGS IN THE SOFTWARE.
 //
 
-#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
```

Comment:
UIKit was not needed.

Also, you'll see many `import UIKit` in the diff.  All those Swift files compiled before without importing UIKit because `SFHFKeychainUtils.h` did that for them and they got access to it via the bridging header.

I find it quite interesting that a seldom visited file had such an impact on unrelate files that did not even use its functionality.